### PR TITLE
Set default tags from empire environment.

### PR DESCRIPTION
**Pull request by brianestlin**

### CODE REVIEW

File: metrics/metrics.go [hunk 1]
```diff
@@ -29,6 +29,11 @@ func SetAppName(appName string) {
 	defaultTags["app"] = appName
 }
 
+// SetProcessName adds a "process:<name>" tag to each metric
+func SetProcessName(processName string) {
+	defaultTags["process"] = processName
```
Comment (v-yarotsky) [inline metrics/metrics.go]: how about `defaultTags["process"] = os.Args[0]`?

**Comment by ejholmes** [comment]
We should probably be using the `empire.app.name`/`empire.process.name` tags. The [metrics doc](https://remind.quip.com/oXa4A9lBjVya) has some conventions. This makes it easier to correlate with everything else (e.g. ELB/AWS metrics)

Eventually we'll introduce a statsd proxy that adds these automatically.

**Comment by brianestlin** [comment]
@v-yarotsky @ejholmes How about this?

File: metrics/metrics.go [hunk 2]
```diff
@@ -24,9 +25,10 @@ type MetricsReporter interface {
 	Close() error
 }
 
-// SetAppName adds a "app:<name>" tag to each metric
-func SetAppName(appName string) {
-	defaultTags["app"] = appName
+// SetEmpireDefaultTags sets default tags reflecting the empire environment to each metric
+func SetEmpireDefaultTags() {
+	defaultTags["empire.app.name"] = os.Getenv("EMPIRE_APPNAME")
+	defaultTags["empire.process.name"] = os.Getenv("EMPIRE_PROCESS")
```
Comment (ejholmes) [inline metrics/metrics.go]: Should probably also add a `container_id` tag using `os.Hostname`.

**Comment by v-yarotsky** [comment]
@brianestlin let's add `container_id` and `empire.app.release`, as per [Eric's doc](https://remind.quip.com/oXa4A9lBjVya) then

**Comment by brianestlin** [comment]
@ejholmes @v-yarotsky I think everything conforms to Eric's doc now, PTAL.

**Pull request merged**